index.js

```diff
@@ -3,30 +3,76 @@
  * @param {number[]} rolls - An array of numbers representing rolls on a die.
  * @returns {boolean} Returns `true` if all values in the array are numbers. Otherwise, return `false`.
  */
-function isValid(rolls) {}
+function isValid(rolls) {
+  let accumulator = false
+  if (rolls.length < 1){
+    accumulator = true
+  }
+  for (const roll of rolls){
+    if (typeof roll === 'number'){
+      accumulator = true
```

Comment:
instead of reassigning your default value every time you hit a number, you can make your code more efficient by starting with it set to true. Therefore, you will only reassign it when the element is not a number